fuzz/meson.build

```diff
@@ -3,12 +3,23 @@ fuzz_src = files(
     'writeout.c',
 )
 
+link_language = 'c'
+fuzz_ldflags = get_option('fuzz-ldflags')
+# Assume that if fuzz_ldflags is set, C++ linking is required.
+# This is necessary for OSS-Fuzz.
+if fuzz_ldflags != ''
+    add_languages('cpp', native: false)
+    link_language = 'cpp'
```

Comment:
i can’t otoh say if there’s any setup needing `fuzz_ldflags` which _doesn't_ always come with a C++ toolchain, but at least a plain, non-OSS-Fuzz libfuzzer setup [needs this option](https://github.com/libass/libass/commit/9987eef44384b368670d962fca57b5994b216499) but is fine with just the C-interface of clang. 

I’d much prefer if such a change wouldn't just always or implicitly magically happen, but instead be an explicit toggle; whether it’s a specific "link fuzzer with C++" toggle or a special-purpose "oss-fuzz" toggle i don’t mind

Reply:
To be honest, the "link with cpp is required" by oss-fuzz is bit hairy. But they established system like that, so I want to follow it to avoid like breakage of build when the do something stupid in env.

Unfortunately it doesn't work, now I know it will break regardless, one way or anther.

I will add an option `fuzz_link_langauge` to go in hand with `fuzz_ldflags`